### DISCUSSION TITLE
Redis database number

### DISCUSSION THREAD

**Discussion by thuan1412** [Budibase/budibase#9458] [Help] [answered]
I don't see any input for configuring the `database number` in the Redis connector.
![image](https://user-images.githubusercontent.com/36019052/215032471-d79f0f20-dc1b-46c0-b1fa-b704d562a57b.png)

Seem like it always uses the default database number (that's 0). 
How can I set a specific database number for the Redis connector?

**Comment by melohagan** [comment, marked as answer]
Hey @thuan1412 

This currently can't be done, but I've raised an issue here: https://github.com/Budibase/budibase/issues/9460

**Reply by thuan1412** [reply]
Thank you!
